### PR TITLE
Add frontend first draft

### PR DESCRIPTION
**Pull request by Andy-Bigler**
This is more or less a one to one copy of the template. In a second setup I would say we clean this up a bit and make it work for our needs in this project. We also need to extend the docker-compose file.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -36,3 +36,23 @@ jobs:
 
     - name: Test
       run: go test -v ./...
+
+  ui-build:
```
Comment (Andy-Bigler) [inline .github/workflows/ci.yaml:40]: We also need to enable linting for the ui

File: ui/app/.huskyrc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "hooks": {
```
Comment (Andy-Bigler) [inline ui/app/.huskyrc:2]: @janfuhrer Maybe you need to explain me how husky works in a sub folder of a git repo that already has hooks. And as far as I understand it the husky setup is a bit different now: https://github.com/typicode/husky

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Andy-Bigler) [reply .github/workflows/ci.yaml:40]: This build steps also needs to be extended

File: ui/app/.huskyrc (hunk 2 above)
Comment (Andy-Bigler) [reply ui/app/.huskyrc]: Removed husky. Everything done we pre commit now.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Andy-Bigler) [reply .github/workflows/ci.yaml:40]: Will be done in the next PR

**Review by janfuhrer — APPROVED** [review]
lgtm

**Pull request merged**